public/static/docs/glossary.js

```diff
@@ -8,33 +8,27 @@ export default {
       name: 'Workspace',
       match: ['workspace'],
       desc: `
-Directory containing all your project files. For example raw datasets, source
-code, ML models, etc. A workspace becomes a **DVC project** when
-[\`dvc init\`](/doc/command-reference/init) is run, and
-[DVC-files](/doc/user-guide/dvc-file-format) or stage files are created in it.
-
-Note that [external outputs](/doc/user-guide/managing-external-data) also
-form part of your expanded workspace, technically.
+Directory containing all your project files, for example raw datasets, source
+code, ML models, etc. It will conatain your DVC project.
       `
     },
     {
       name: 'DVC Project',
-      match: ['DVC project', 'project', 'projects'],
+      match: [
+        'DVC project',
+        'project',
+        'projects',
+        'DVC repository',
+        'DVC repositories'
+      ],
       desc: `
 Initialized by running \`dvc init\` in the **workspace** (typically in a Git
 repository). It will contain the
 [\`.dvc/\` directory](/doc/user-guide/dvc-files-and-directories) and
 [DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
-\`dvc add\` or \`dvc run\`. It may also be a Git repository.
-      `
-    },
-    {
-      name: 'DVC Repository',
-      match: ['DVC repository', 'DVC repositories'],
-      desc: `
-**DVC project** initialized using \`dvc init\` in a Git repository. It will
-contain \`.git/\` and [\`.dvc/\`](/doc/user-guide/dvc-files-and-directories)
-directories, as well as any DVC-files created by DVC.
+\`dvc add\` or \`dvc run\`. Some DVC commands require it to be in a Git repo –
+explained in each [command reference](/doc/command-reference) but usually
+obvious from the context.
       `
     },
```

Comment:
These terms were merged per https://github.com/iterative/dvc.org/pull/931#pullrequestreview-351903751, which was the right move. But after a short discussion in https://github.com/iterative/dvc/pull/3277#discussion_r374425536 I think maybe we should make "DVC repository" (or just repo) the main or only term?

Initially I only thought of repos as the Git part of DVC projects, but later on the term DVC repo started popping up and we even had its glossary entry, now we've merged them both but left "project" as the main term. Comparing vs the core DVC repo though, the term "project" is not used at all so it would be easier to just manage the single term "repo" even when sometimes it will be necessary to disambiguate between Git vs DVC – a problem we currently have anyway.

Thoughts @shcheklein ? I can do this in my next regular updates or in its own PR (if it affects lots of files) if so.